docs/Training-ML-Agents.md

```diff
@@ -117,6 +117,13 @@ Python by using both the `--resume` and `--inference` flags. Note that if you
 want to run inference in Unity, you should use the
 [Unity Inference Engine](Getting-Started.md#running-a-pre-trained-model).
 
+Additionally, if the network architecture changes, you may still load an existing model,
+and ML-Agents will only load the parts of the model that haven't changed. For instance,
```

Comment:
This is ambiguous because the user does not know which parts would change. Adding a new reward signal requires the critic to be re-initialized for example.

Reply:
```suggestion
but ML-Agents will only load the parts of the model it can load and ignore all others. For instance,
```
This is still ambiguous but the user will not try to interpret what "it can load" means (while they will try to interpret "the parts that haven't changed")